README.md

```diff
@@ -1,2 +1,70 @@
+| [notation-core-go](https://github.com/notaryproject/notation-core-go) | This repository contains the source code for the Golang library implementation of the Notary signature specification and wrapping (COSE and JWS). |
+| [roadmap](https://github.com/notaryproject/roadmap) | This repository is intended for keeping track of development activitis in the Notary project. It may be retired in the future. |
+| [tuf](https://github.com/notaryproject/tuf) | This repository is intended for prototyping the storage of TUF metadata in OCI-compliant registries. It is not under active development at the moment but there are plans to revive it in the future. |
+
+## Project Status
+
+The Notary Project is in active development. The latest release announcement are published on the [Notary Project blog](https://notaryproject.dev/blog/). The Notary community uses the [project board](https://github.com/orgs/notaryproject/projects/10) for project planning and status tracking. You can also use GitHub milestones to track the progress of each repository:
+
+- [The Notary Project specification milestones](https://github.com/notaryproject/notaryproject/milestones)
+- [notation CLI milestones](https://github.com/notaryproject/notation/milestones)
+- [notation-go library milestones](https://github.com/notaryproject/notation-go/milestones)
+- [notation-core-go library milestones](https://github.com/notaryproject/notation-core-go/milestones)
+- [notary milestones](https://github.com/notaryproject/notary/milestones)
+
+You can also check the release pages of each repository for the laste release binaries:
+
+- [The Notary Project specification releases](https://github.com/notaryproject/notaryproject/releases)
+- [notation CLI releases](https://github.com/notaryproject/notation/releases)
+- [notation-go library releases](https://github.com/notaryproject/notation-go/releases)
+- [notation-core-go library releases](https://github.com/notaryproject/notation-core-go/releases)
+- [notary releases](https://github.com/notaryproject/notary/releases)
+
+## Security
+
+The Notary Project has had several public security audits:
+
+- [July 31, 2015 by NCC](https://github.com/notaryproject/notary/blob/master/docs/resources/ncc_docker_notary_audit_2015_07_31.pdf) covering `notary` repository.
+- [August 7, 2018 by Cure53](https://github.com/notaryproject/notary/blob/master/docs/resources/cure53_tuf_notary_audit_2018_08_07.pdf)) covering TUF and the `notary` repository.
+- [Mar 21, 2023 by ADA Logics](https://github.com/notaryproject/notaryproject/blob/main/security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf) fuzz testing audit covering `notary`, `notation-go`, and `notation-core-go` repositories.
+
+The Notary Project has a continuous fuzz testing implemented for the following repositories: `notary`, `notation-go`, and `notation-core-go`.
```

Comment:
Should we also add the first fuzz testing report or blog to this sentence to provide context reference? https://www.cncf.io/blog/2023/03/21/the-notary-project-completes-fuzzing-security-audit/

Reply:
We link to the PDFs of the reports. I believe this is sufficient for the README.